dev/server/src/index.ts

```diff
@@ -32,7 +33,7 @@ async function start (): Promise<void> {
 
   const db = client.db('accounts')
 
-  await upgradeWorkspace(defaultWorkspace, { mongoDBUri: dbUri })
+  await upgradeWorkspace(defaultWorkspace, { mongoDBUri: dbUri, txes: builder.getTxes() })
```

Comment:
It gives me error:
 TS2345: Argument of type '{ mongoDBUri: string; txes: Tx<Doc>[]; }' is not assignable to parameter of type 'WorkspaceOptions'.   Object literal may only specify known properties, and 'txes' does not exist in type 'WorkspaceOptions'.

Reply:
Please do `rush build`